3party/sdf_image/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 project(sdf_image)
 
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(
```

Comment:
You didn't get my idea. I see a lot of code duplication here:
```
if (CMAKE_CXX_COMPILER_ID MATCHES "CLang")
  add_compile_options( ...)
endif()
```
I suggest you to create a macro and get rid of code duplication.